ompi/mpi/c/init_thread.c.in

```diff
@@ -39,22 +40,32 @@
 PROTOTYPE ERROR_CLASS init_thread(INT_OUT argc, ARGV argv, INT required,
                                   INT_OUT provided)
 {
-    int err, safe_required = MPI_THREAD_SERIALIZED;
+    int err, err_arg_required = false, safe_required = MPI_THREAD_SERIALIZED;
```

Comment:
If `err_arg_required` is going to only take on boolean values and be used as a boolean value, it should be of type `bool`.